requirements.txt

```diff
@@ -29,3 +29,4 @@ reportlab >=3.3.0, <4
 requests~=2.21.0
 six~=1.12.0
 unittest-xml-reporting~=2.5.1
+kubernetes==12.0.1
```

Comment:
We could move this to the `requirements_prod.txt` file since it is not needed in local testing in the apluslms/run-aplus-front container.